app/controllers/two_factor_authentication/recovery_code_controller.rb

```diff
@@ -3,11 +3,28 @@ class RecoveryCodeController < ApplicationController
     before_action :confirm_two_factor_authenticated
 
     def show
-      @code = RecoveryCodeGenerator.new(current_user).create
+      @code = create_new_code
     end
 
     def acknowledge
       redirect_to after_sign_in_path_for(current_user)
     end
+
+    private
+
+    def create_new_code
+      generator = RecoveryCodeGenerator.new(current_user)
+      code = generator.create
+      reencrypt_pii_recovery(generator.user_access_key) if
+        current_user.active_profile.present?
```

Comment:
well rubocop is smarter than I thought. fixing multiline conditionals as I find them.